test/e2e/argo-workflows/agent.yaml

```diff
@@ -164,6 +164,43 @@ spec:
                   path: /var/run/dogstatsd
                 name: dogstatsd
 
+    - name: logs-hello-world
+      value: |
+        apiVersion: extensions/v1beta1
+        kind: Deployment
+        metadata:
+          name: logs-hello-world
+          namespace: default
+        spec:
+          replicas: 1
+          template:
```

Comment:
How to we tell the agent to collect these logs ? there's no annotation & no `collect_all` 🤔